Feature/add logging

santiagonoguez opened the pull request:
Logging functionality added, along with status checking to ensure only logging changes are sent as discussed in proposal.
Also manually tested logging in newly created BQ table.

tejohnso commented on src/presence.js:
```diff
@@ -0,0 +1,35 @@
+
+  Object.assign(statusTable, updatedStatusTable);
+
+  return logger.all("watching").then(() =>
```
Isn't this going to result in a "watching" entry in BQ every time a module sends its heartbeat?  I don't think that would be necessary. Where is updateStatus being called?

santiagonoguez replied:
@tejohnso my idea is it will be called with the full updated status from all modules ( next card ), so "watching" is only logged every 5 minutes, not with every heartbeat update. So every 5 minutes the updated current known status for all modules is passed as argument to this function and the problem you describe does not happen.

santiagonoguez commented:
@tejohnso I think part the confusion was in the function and field names, I've renamed updateStatus() to updateStatusTable() to make it clear that we are updating the whole set, and also renamed statusTable to previousStatusTable to make it clear that we are referring to the last logged values.
